### ISSUE TITLE
add output screenshot in readme

### ISSUE THREAD

**Issue by vanshg395** [smartnsoft/flappy_search_bar#43]
I think if you put ui screenshots or gifs, it would be better so that I can see how it works before adding to my project.

**Comment by vinceramcesoliveros** [comment]
Yes. Similar to this https://pub.dev/packages/material_floating_search_bar
